docs/concepts/unions.md

```diff
@@ -285,10 +288,18 @@ class DiscriminatedModel(BaseModel):
 
 data = {'x': {'x': {'x': 'a'}}}
 m = DiscriminatedModel.model_validate(data)
-assert m == (
-    DiscriminatedModel(x=DiscriminatedModel(x=DiscriminatedModel(x='a')))
-)
-assert m.model_dump() == data
+print(m)
```

Comment:
I believe it's most reader friendly to use print than assert, can we check were else we're using `assert`.

Reply:
Agreed. I'll change the rest of the `assert` statements to `print` statements.